test/colorado_lottery_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/contestant'
+require './lib/game'
+require './lib/colorado_lottery'
+require 'pry'
+
+class ColoradoLotteryTest < Minitest::Test
+  def test_it_exists
+    lottery = ColoradoLottery.new
+    assert_instance_of ColoradoLottery, lottery
+  end
+
+  def test_it_has_registered_contestants
+    lottery = ColoradoLottery.new
+    #unable to resove error with testing
+    assert_equal {} , lottery.registered_contestants
```

Comment:
😢 Yea, this is a bit tricky - when you need to assert any sort of hash, you need to either wrap the hash in parens `({})`, use `Hash.new`, or create an expected variable that stores the hash.